go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/CodeChain-io/codechain-rpc-go
 
 go 1.12
+
+require github.com/ethereum/go-ethereum v1.9.6
```

Comment:
go ethereum is a very huge project.
Can't we import just the rlp package?

Reply:
I'm looking for importing only rlp package but it is hard to find.

Reply:
OK. Let's fix it later.

Reply:
I added issue #13.